src/Api/Controller/ListPostsController.php

```diff
@@ -41,7 +41,7 @@ class ListPostsController extends AbstractListController
     /**
      * {@inheritdoc}
      */
-    public $sortFields = ['createdAt'];
+    public $sortFields = ['number'];
```

Comment:
This should probably still be supported, in case someone wants to sort by createdAt from an external API.

Reply:
```suggestion
    public $sortFields = ['number', 'createdAt'];
```